Makefile

```diff
@@ -166,6 +166,11 @@ else ifdef DEPLOY_MANIFEST_TAG
   DEPLOY_TAG_OPTION = --deploy-manifest-tag "$(DEPLOY_MANIFEST_TAG)"
 endif
 
+define restart_service_pods
+$(KUBECTL) rollout restart deployment assisted-service
+$(KUBECTL) rollout status  deployment assisted-service
```

Comment:
is that for debug?

Reply:
I think it is meant for reloading the assisted-service config and apply:
`AUTH_TYPE=none ENABLE_KUBE_API=true`

Reply:
Looks like it monitor the status of the rollout, please add a comment,